backend/src/utils/users.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+    validateUserObj: (req, res, next) => {
```

Comment:
In general, the validations should not have access to req, res and next. Instead they should be simple functions that either return nothing in case the validation succeeded or an error string in case there was a problem. This way they are easy to test and the logic for responses is contained in the routing files themselves.